src/library_pthread.js

```diff
@@ -619,11 +701,25 @@ var LibraryPThread = {
     PThread.pthreads[threadParams.pthread_ptr] = worker;
 
     worker.pthread_ptr = threadParams.pthread_ptr;
+
+    var MsgChannel;
+#if ENVIRONMENT_MAY_BE_NODE
+    if (ENVIRONMENT_IS_NODE) {
+      // TODO: This isn't necessary in Node 18+
+      MsgChannel = require('worker_threads').MessageChannel;
+    } else {
+      MsgChannel = MessageChannel;
+    }
+#else
+    MsgChannel = MessageChannel;
+#endif
```

Comment:
Can we avoid declaring a new name here and just polyfil on node using `var MessageChannel = require ...`?

Reply:
(or `global.MessageChannel = require ...`)